app/views/groups/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1><%= @group.name %></h1>
+
+<h2>Owners</h2>
```

Comment:
Could you internationalize this?

Reply:
Should be done. I didn't know how to translate "Owners", I found "Besitzer" do sound a bit weird. Ended up translating as "Admins"